src/SFA.DAS.EmployerFinance.Web/Orchestrators/TransfersOrchestrator.cs

```diff
@@ -39,19 +44,20 @@ public class TransfersOrchestrator
         await Task.WhenAll(indexTask, accountDetail);
 
         Enum.TryParse(accountDetail.Result.ApprenticeshipEmployerType, true, out ApprenticeshipEmployerType employerType);
-
+       
         return new OrchestratorResponse<IndexViewModel>
         {
             Data = new IndexViewModel
             {
                 IsLevyEmployer = employerType == ApprenticeshipEmployerType.Levy,
                 PledgesCount = indexTask.Result.PledgesCount,
                 ApplicationsCount = indexTask.Result.ApplicationsCount,
-                RenderCreateTransfersPledgeButton = renderCreateTransfersPledgeButton,                    
+                RenderCreateTransfersPledgeButton = renderCreateTransfersPledgeButton,
                 StartingTransferAllowance = accountDetail.Result.StartingTransferAllowance,
                 FinancialYearString = DateTime.UtcNow.Year.ToString(),
                 HashedAccountID = hashedAccountId,
                 CurrentYearEstimatedSpend = indexTask.Result.CurrentYearEstimatedCommittedSpend,
+                MinimumTransferFunds = _configuration.MinimumTransferFunds
```

Comment:
I think a better approach to this, since the minimum amount is unlikely to ever change, would be to hardcode it as a constant in this orchestrator. Then, in this method, simply check the EstimatedRemainingAllowance (it might be worth removing the calculation for this from the ViewModel and putting it here) against the minimum and, if applicable, set a new property, say:

```
bool EstimatedRemainingAllowance { get; set; }
```

to false. That way, the logic (though simple) is all in the orchestrator and can be unit tested, and the view markup is simplified as it merely has to check this bool.